tests/python/unittest/test_loss.py

```diff
@@ -143,8 +143,9 @@ def test_kl_loss():
             eval_metric=mx.metric.Loss(), optimizer='adam')
     assert mod.score(data_iter, eval_metric=mx.metric.Loss())[0][1] < 0.05
 
-
-@with_seed(1234)
+# @eric-haibin-lin: Getting rid of fixed seed,
+# tracked at https://github.com/apache/incubator-mxnet/issues/11699
```

Comment:
I would prefer to add this in the PR description instead of code, since this issue # may change over time. This makes code less clean.